spyder_kernels/customize/code_runner.py

```diff
@@ -145,10 +145,11 @@ class SpyderCodeRunner(Magics):
     def __init__(self, *args, **kwargs):
         self.show_global_msg = True
         self.show_invalid_syntax_msg = True
+        super().__init__(*args, **kwargs)
```

Comment:
```suggestion
        super().__init__(*args, **kwargs)

        self.show_global_msg = True
        self.show_invalid_syntax_msg = True
```

It should be possible to call `super().__init__` at the beginning of this method, which is customary and makes the code easier to read.